client/src/components/HtmlEditorField/HtmlEditorField.js

```diff
@@ -0,0 +1,46 @@
+/* global window */
+import React from 'react';
+import Script from 'react-load-script';
+import { Component as TextField } from 'components/TextField/TextField';
+import fieldHolder from 'components/FieldHolder/FieldHolder';
+
+class HtmlEditorField extends TextField {
+  constructor(props) {
+    super(props);
+    this.state = {
+        isReady: false
+    };
+    this.handleReady = this.handleReady.bind(this);
+  }
+
+  getInputProps() {
+    return {
+      ...super.getInputProps(),
+      ...this.props.data.config,
+    };
+  }
+
+  handleReady() {
+    if (!window.TinyMCE && window.tinymce) {
+      window.TinyMCE = window.tinymce;
+    }
+    this.setState({ isReady: true });
+  }
+
+  renderDependencyScript() {
+    return <Script url={this.props.data.editorjs} onLoad={this.handleReady} />;
```

Comment:
this is interesting, because when I had a go at implementing this, I ran into issues of the script loading multiple times if I had multiple instances of this rendered and on a slow connection.
Have you tested this out? Do you know if load-script manages that?

Reply:
I will investigate.

Reply:
No, it is safe :)

I tested in browser and it seemed fine.
Investigation into why revealed:
https://github.com/blueberryapps/react-load-script/blob/master/src/index.jsx#L46